quinn-h3/src/lib.rs

```diff
@@ -159,27 +173,52 @@ impl From<proto::headers::Error> for Error {
     }
 }
 
+/// Errors defined by the HTTP/3 protocol
+///
+/// Read the [`HTTP/3 specification`] for more details.
+///
+/// [`HTTP/3 specification`]: https://quicwg.org/base-drafts/draft-ietf-quic-http.html#name-http-3-error-codes
 #[derive(Debug)]
 pub enum HttpError {
+    /// No error. This is used when the connection or stream needs to be closed, but there is no error to signal.
```

Comment:
Remove the periods on these comments, please? In the first one, maybe replace the first period with `;`.